contrib/install_db4.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Install libdb4.8 (Berkeley DB).
+
+set -e
+
+if [ -z "${1}" ]; then
+  echo "Usage: ./install_db4.sh <base-dir> [<extra-bdb-configure-flag> ...]"
+  echo
+  echo "Must specify a single argument: the directory in which db5 will be built."
+  echo "This is probably \`pwd\` if you're at the root of the bitcoin repository."
+  exit 1
+fi
+
+BDB_PREFIX="${1}/db4"; shift;
+BDB_EXTRA_CONFIGURE_FLAGS="${@}"
+BDB_VERSION='db-4.8.30.NC'
+BDB_HASH='12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef'
+BDB_URL="https://download.oracle.com/berkeley-db/${BDB_VERSION}.tar.gz"
+
+mkdir -p $BDB_PREFIX
+
+wget --no-check-certificate "${BDB_URL}" -O "${BDB_VERSION}.tar.gz"
```

Comment:
would be great if we could have a `curl --insecure` fallback if wget is not present. IMO `curl` should be the default nowadays.